docs/DEPLOY-SMART-CONTRACT.md

```diff
@@ -0,0 +1,54 @@
+## 1. DEPLOY SMART-CONTRACT
+
+Fill in the `governor` wallet private key and address.
+
+```
+# Deploy Contract
+DEPLOY_OWNER_ADDRESS=
+DEPLOY_PRIVATE_KEY=
+
+```
+
+Then it's able use a already contract deployed or deploy a new one, in this step pay attention because the **order to run database migrations** is differente in each option.
```

Comment:
```suggestion
Then will be able to use an already deployed contract or deploy a new one, in this step pay attention because the **order to run database migrations** is differente in each option.
```